README.md

```diff
@@ -1,33 +1,38 @@
 # another-go-installer
 
-Script to install `Golang` and create a basic [workspace](https://golang.org/doc/code.html). By default the script takes the lates version in the download [page](https://golang.org/dl/)
+Script to install `Golang` and create a basic [workspace](https://golang.org/doc/code.html#Workspaces). By default the script takes the latest version in the download [page](https://golang.org/dl/).
 
 ## Features
 
 ✔ Detects automatically the Operative System (`mac`, `linux`)
 
 ✔ Detects automatically the architecture of your OS (`64`, `32` bits)
 
-❌ Prompt for a github user (to create projects directory inside the workspace)
+❌ Prompts for a github user (to create projects directory inside the workspace)
 
-❌ Install a specific version
+❌ Installs a specific version
 
 ---
```

Comment:
Now that the `usage` and `options` are headers this line can be removed because the lvl 2 headers have their own underline 😬

Reply:
I will also remove [the one above `See also`](https://github.com/cabaalexander/another-go-installer/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R30) since its a header too